### DISCUSSION TITLE
How much we can contribute?

### DISCUSSION THREAD

**Discussion by paoloricciuti** [CodingGarden/listd#41] [Q&A] [answered]
For the moment i was limiting my self to fix bugs because i tought @w3cj would actually like to develop feature live on twitch.

How much we can go on to contribute with new features?

**Comment by omar2205** [comment]
We still setting up the project, so CJ will have to create some issues like FreshSpots

**Comment by w3cj** [comment, marked as answer]
Right now I want to limit contributions to bug fixes and configuration fixes. I know @omar2205 has #38 open which is technically a new feature but it has revealed a config issue that I would like to resolve before the next stream.

If you have a feature you want to work on let me know here in discussions or in issues and I can let you know if it is something I am okay with not doing myself on stream.

**Reply by paoloricciuti** [reply]
> Right now I want to limit contributions to bug fixes and configuration fixes. I know @omar2205 has #38 open which is technically a new feature but it has revealed a config issue that I would like to resolve before the next stream.

Yeah that's what i tought...obviously there's no problem in adding the feature from @omar2205 : this question was obviously inspired by that pr but it was not meant to be "rude". I was just curious if i could start to also develop some features, nothing more. I'll stick with my decision of just bug fixes than 😄 

> If you have a feature you want to work on let me know here in discussions or in issues and I can let you know if it is something I am okay with not doing myself on stream.

Nothing in particular for the moment...just want to give a hand ✋

**Reply by w3cj** [reply]
Sounds good. Eventually I want to have all of the upcoming features as open issues  and on the project board. I think at that point any open issue will be open to anyone to contribute.

**Reply by paoloricciuti** [reply]
Perfect!

**Reply by ToxicToast** [reply]
> Sounds good. Eventually I want to have all of the upcoming features as open issues  and on the project board. I think at that point any open issue will be open to anyone to contribute.

Is there a Board yet with all open issues?
